src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -931,29 +931,42 @@ protected virtual async Task<bool> ShouldSkipMethod(SessionId sessionId, Executi
             if (shouldReturn)
                 return true;
 
-            if (j == 0 && method?.Info?.DebuggerAttrInfo?.DoAttributesAffectCallStack(JustMyCode) == true)
+            if (j == 0)
```

Comment:
For a future PR: rename `j` to `frameNumber` or something appropriate.

Reply:
just a nit: We could change this to `if (j != 0) return false;`.

Reply:
Um in fact, can this check be moved to the first line of this method?

Reply:
done :)